modules/apps/foundation/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectServiceHandlerImpl.java

```diff
@@ -101,7 +101,7 @@
 	public boolean hasValidOpenIdConnectSession(HttpSession httpSession)
 		throws OpenIdConnectServiceException.NoOpenIdConnectSessionException {
 
-		OpenIdConnectSession openIdConnectSession = getOpenIdConnectSession(
```

Comment:
This should be unchanged since we should use the interface instead of the impl (except for when we need to create a new instance). We can also use the impl for cases where we need something specific to nimbusds (eg if it needs the AccessToken objects or so on for session expiration/validation/etc).